tests/test_schema.py

```diff
@@ -459,26 +460,17 @@ def test_json_schema():
     class Model(BaseModel):
         a = b'foobar'
         b = Decimal('12.34')
+        c = date(2022, 8, 11)
```

Comment:
This isn't testing `condate`! hence the coverage is not 100%

Reply:
sorry I got the wrong idea about what to test.  think I've got it now, will push shortly